cmd/rootCmd.go

```diff
@@ -96,26 +98,26 @@ func Baseline(ctx context.Context, output string) {
 	}
 	fmt.Println("Available Storage Provisioners:")
 	fmt.Println()
-	time.Sleep(500 * time.Millisecond)
+	time.Sleep(500 * time.Millisecond) // Added to introduce lag.
 	for _, provisioner := range provisionerList {
 		provisioner.Print()
 		fmt.Println()
 		time.Sleep(500 * time.Millisecond)
 	}
 }
 
-// // Fio executes the FIO test.
-// func Fio(output, storageclass string) {
-// 	p := kubestr.NewKubestr()
-// 	result := p.Baseline()
-// 	if output == "json" {
-// 		jsonRes, _ := json.MarshalIndent(result, "", "    ")
-// 		fmt.Println(string(jsonRes))
-// 		return
-// 	}
-// 	for _, retval := range result {
-// 		retval.Print()
-// 		fmt.Println()
-// 	}
-// 	return
-// }
+// Fio executes the FIO test.
+func Fio(ctx context.Context, output, storageclass string) {
+	p, err := kubestr.NewKubestr()
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	result := p.FIO(ctx, storageclass)
+	if output == "json" {
+		jsonRes, _ := json.MarshalIndent(result, "", "    ")
+		fmt.Println(string(jsonRes))
+		return
+	}
+	result.Print()
+}
```

Comment:
What linter are you using? I know that this would not pass golangci with wsl, whitespace etc.

Reply:
Yea lint failed.

Reply:
But I don't know how? It looks ok in vscode, not sure why its not the same here